controllers/apply/under10k/fields/project-dates-next.js

```diff
@@ -0,0 +1,249 @@
+        'england': 18,
+        'northern-ireland': 12,
+        'scotland': 12,
+        'wales': 12,
+    };
+
+    return countryLeadTimes[country] || 18;
+}
+
+module.exports = {
+    _getLeadTimeWeeks: getLeadTimeWeeks, // Exported for tests
+
+    fieldProjectStartDateCheck(locale, data = {}) {
+        const localise = get(locale);
+
+        const projectCountry = get('projectCountry')(data);
+        const supportingCOVID19 = get('supportingCOVID19')(data);
+
+        function options() {
+            const optionAsap = {
+                value: 'asap',
+                label: localise({
+                    en: `As soon as possible`,
+                    cy: `@TODO: i18n`,
+                }),
+            };
+
+            const optionExactDate = {
+                value: 'exact-date',
+                label: localise({
+                    en: `Enter an exact date`,
+                    cy: `@TODO: i18n`,
+                }),
+            };
+
+            if (projectCountry === 'england' || supportingCOVID19 === 'yes') {
+                optionExactDate.attributes = { disabled: 'disabled' };
+            }
+
+            return [optionAsap, optionExactDate];
+        }
+
+        return new RadioField({
+            locale: locale,
+            name: 'projectStartDateCheck',
+            label: localise({
+                en: `When would you like to get the money if you are awarded?`,
+                cy: `@TODO: i18n`,
+            }),
+            options: options(),
+            messages: [
+                {
+                    type: 'base',
+                    message: localise({
+                        en: 'Select an option',
+                        cy: 'Dewis opsiwn',
+                    }),
+                },
+            ],
+        });
+    },
+    fieldProjectStartDate(locale, data) {
+        const localise = get(locale);
+
+        const projectCountry = get('projectCountry')(data);
+
+        const minDate = moment().add(getLeadTimeWeeks(projectCountry), 'weeks');
+
+        const minDateExample = minDate
+            .clone()
+            .locale(locale)
+            .format('DD MM YYYY');
+
+        function schema() {
+            /**
+             * When projectStartDateCheck is asap
+             * we don't show the project start date question
+             * and instead pre-fill it with the current date
+             * at the point of submission (see forSalesforce())
+             */
+            return Joi.when('projectStartDateCheck', {
+                is: 'asap',
+                then: Joi.any().strip(),
+                otherwise: Joi.dateParts()
+                    .minDate(minDate.format('YYYY-MM-DD'))
+                    .required(),
+            });
+        }
+
+        return new DateField({
+            locale: locale,
+            name: 'projectStartDate',
+            label: localise({
+                en: `Tell us when you'd like to get the money if you're awarded funding?`,
```

Comment:
Again on the wording here, it feels a bit weird grammatically to have a question start with "Tell us". Maybe "When would you like to receive the money if you're awarded funding?".